condor_watch_q.py

```diff
@@ -37,7 +37,14 @@
 
 
 def parse_args():
-    parser = argparse.ArgumentParser(
+    class CustomParser(argparse.ArgumentParser):
+        def error(self, message):
+            args, unknown = parser.parse_known_args()
+            parse_unknown_args(unknown)
+            self.print_usage()
+            sys.exit(1)
```

Comment:
Instead of doing this manually, I recommend just calling up to the superclass. They do some special behavior there that we should inherit.

```suggestion
            super(self, CustomParser).error(message)
```